lib/glob.js

```diff
@@ -1,12 +1,19 @@
 'use strict';
 
-var braces = require('braces');
-var brackets = require('expand-brackets');
-var extglob = require('extglob');
-var parse = require('parse-glob');
+var isExtglob = require('is-extglob');
 var chars = require('./chars');
 
 /**
+ * Lazily required module dependencies.
+ */
+
+var lazy = require('lazy-cache')(require);
+var lazyBraces = lazy('braces');
```

Comment:
oh nevermind, I see the distinction now - you have to call the returned function to actually initialize the module... I had assumed it was like a seamless wrapper